### PR TITLE
Remove redundant feature flags

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-2449

### CODE REVIEW

**Review by maciekzygmunt — APPROVED** [review]

**Review by maciekzygmunt — APPROVED** [review]

File: packages/frontend/src/build/config/common.ts [hunk 1]
```diff
@@ -20,17 +20,13 @@ export const common: Omit<Config, 'backend'> = {
     banner: new Date() <= GITCOIN_BETA_ROUND_END,
     gitcoinOption: false,
     hiring: true,
-    hiringBadge: false,
+    hiringBadge: true,
```
Comment (sz-piotr) [inline packages/frontend/src/build/config/common.ts:23]: Why this change?

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/frontend/src/build/config/common.ts (hunk 1 above)
Comment (torztomasz) [reply packages/frontend/src/build/config/common.ts:23]: Oh, my mistake, I thought I discarded it.

**Review by sz-piotr — APPROVED** [review]
